configfile.js

```diff
@@ -266,6 +266,7 @@ cfreader.ensure_enoent_timer = function () {
             })(files[i]); // END BLOCK SCOPE
         }
     }, 60 * 1000);
+    cfreader._enoent_timer.unref();
```

Comment:
BTW how come we now have a haraka-config package and this file is still in Haraka?